testapp/src/main/res/values/strings.xml

```diff
@@ -63,4 +64,12 @@
     <string name="tut_setting">This is the first time you launch the MiniApp Demo application on this device. You need to provide your own Host Project ID and Subscription Key from &lt;font color="#BF0000">&lt;b>Rakuten&lt;/b>&lt;/font> &lt;b>App Studio&lt;/b></string>
 
     <string name="prefix_placeholder">%1$s %2$s</string>
+
+    <string name="miniappdemo_scheme">miniappdemo</string>
+    <string name="miniappdemo_host">miniapp</string>
+    <string name="miniappdemo_pathprefix">/preview/token/</string>
+
+    <string name="perm_desc_A_PERMISSION">test permission</string>
+    <string name="perm_label_A_PERMISSION">test permission</string>
```

Comment:
I think these aren't being used for anything, right?

Reply:
oh yes..i tested custom permission but forget to remove it.. sorry.. i will remove them